Purchases/Public/Package.swift

```diff
@@ -96,16 +108,16 @@ private extension PackageType {
 @objc public extension Package {
 
     /**
-     * - Parameter packageType: A PackageType.
+     * - Parameter packageType: A `PackageType`.
      * - Returns: an optional description of the packageType.
      */
     static func string(from packageType: PackageType) -> String? {
         return packageType.description
     }
 
     /**
-     * - Parameter string: A string that maps to a enumeration value of type PackageType
-     * - Returns: a PackageType for the given string.
+     * - Parameter string: A string that maps to a enumeration value of type `PackageType`
+     * - Returns: a `PackageType` for the given string.
```

Comment:
I think we should try to do double-backticks for all type references, that way [DocC will auto-link them to the respective types](https://developer.apple.com/documentation/xcode/formatting-your-documentation-content#Link-to-Symbols-and-Other-Content). This is [compatible with Jazzy too](https://github.com/realm/jazzy#supported-documentation-keywords)

Reply:
We can do it in a follow-up pass too

Reply:
Done!